README.hpcgap.md

```diff
@@ -0,0 +1,14 @@
+# HPC-GAP
+
+The beta release of GAP 4.9 includes experimental code to support multithreaded
+programming in GAP. GAP and HPC-GAP codebases diverged during the project, and
+we are currently working on unifying the codebases and incorporating the HPC-GAP
+code back into the mainstream GAP versions.
+
+This is work in progress, but including it into GAP 4.9 release will allow to
+conduct further code refactoring more efficiently, and will expose some new
+code to make visible new names and keywords. 
+
+Users interested in experimenting with shared memory parallel programming
+in GAP are advised to check out the GAP repository from GitHub and follow
+instructions from https://github.com/gap-system/gap/wiki/Building-HPC-GAP.
```

Comment:
Perhaps:
> Users interested in experimenting with shared memory parallel programming in GAP can build HPC-GAP by following the instructions from https://github.com/gap-system/gap/wiki/Building-HPC-GAP. While you can build HPC-GAp from a release version of GAP you downloaded from the GAP website, due to the ongoing development of HPC-GAP, we recommend that you instead build HPC-GAP from the latest development version available in the GAP repository at GitHub, i.e. https://github.com/gap-system/gap

Reply:
Also, somebody should double check those build instructions are up-to-date (perhaps @ssiccha would be willing to help).